### ISSUE TITLE
Bootstrap hardcodes vs2012 toolchain

### ISSUE THREAD

**Issue by benvanik** [premake/premake-core#389]
In Bootstrap.mak:
```
.\build\bootstrap\premake_bootstrap --to=build/bootstrap vs2012
```

This prevents users who do not have the vs2012 SDK (v110) installed from running the bootstrap. Visual Studio 2015 installs don't have the 2012 SDK, and there's no way to install it. Switching that line to use vs2015 will fix things. Would it be possible to have the bootstrap detect the latest VS version or allow this to be passed in as a make option?

**Comment by samsinsane** [comment]
The next line `devenv .\build\bootstrap\Premake5.sln /Upgrade` is supposed to have VS upgrade the project if required, this should work regardless of whether you have VS2012 or not. This might have changed with VS2015? But I haven't encountered this issue yet.

**Comment by benvanik** [comment]
I just upgraded to VS2015 Update 1, perhaps that is the issue. 2015 is the only copy I have installed, so devenv only points to that. I'll wipe my temp and try again.

**Comment by samsinsane** [comment]
Interesting, I just ran the bootstrap process on my machine with VS2015 Update 1 and it didn't perform the upgrade properly. I can't say I'm surprised, I've never had any luck using the upgrade functionality in VS, it always fails to do something and the log doesn't indicate that anything is wrong with the projects at all.

**Comment by benvanik** [comment]
Interestingly an older commit upgrades and builds ok, so it may be a recent (~few months) change that is causing the issue.

For reference, the bootstrap prints out this:
```
        devenv .\build\bootstrap\Premake5.sln /Upgrade

Microsoft Visual Studio 2015 Version 14.0.24720.0.
Copyright (C) Microsoft Corp. All rights reserved.

Migration completed successfully, but some warnings were detected during migration.
For more information, see the migration report:
D:\Dev\xenia\third_party\premake-core\build\bootstrap\UpgradeLog.htm
```

UpgradeLog.htm, but renamed due to github upload restrictions:
[UpgradeLog.txt](https://github.com/premake/premake-core/files/75432/UpgradeLog.txt)

**Comment by samsinsane** [comment]
This was fixed in #404. Use `nmake -f Bootstrap.mak MSDEV=vs2015 windows` to override VS version to vs2015.
